app/src/main/res/layout/fragment_home.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".view.home.HomeFragment">
+
+    <androidx.appcompat.widget.SearchView
+        android:id="@+id/search_bar"
```

Comment:
Let's be consistent with our view id naming.
maybe we can follow below convention `RecyclerView` -> `rv_XXX`

then this will be like `sv_search`